client/app/scripts/superdesk-desks/aggregate.js

```diff
@@ -468,7 +496,7 @@
                             });
                         }
                     });
-
+                    scope.allSavedSearches = false;
```

Comment:
Maybe it would be better to rename this flag to `allSearchesSaved`, because the name `allSavedSearches` could be understood as _"list of all saved searches"_ (or maybe it's just me, a non-native speaker :smile: ).